packages/common/migrations/0001_abnormal_gressill.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TYPE "submission_state" ADD VALUE 'OPEN';--> statement-breakpoint
+ALTER TYPE "submission_state" ADD VALUE 'VALID';--> statement-breakpoint
+ALTER TYPE "submission_state" ADD VALUE 'INVALID';--> statement-breakpoint
+ALTER TYPE "submission_state" ADD VALUE 'CLOSED';--> statement-breakpoint
+ALTER TYPE "submission_state" ADD VALUE 'COMMITED';
```

Comment:
may also need to check the spelling elsewhere
```suggestion
ALTER TYPE "submission_state" ADD VALUE 'COMMITTED';
```